### ISSUE TITLE
Gogs webhook form missing ContentType field

### ISSUE THREAD

**Issue by russell** [go-gitea/gitea#9504]
- Gitea version (or commit ref): 1.11.0+dev-496-g3e166bd05
- Git version:
- Operating system: Linux (docker) docker.io/gitea/gitea@sha256:19fd040118804252e4d991b8279d84be965c12ab1fa02cbb51119e5475b69bb0
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [X] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [X] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

There seems to be a field for content type missing from the page, when i submit the form to add a Gogs webhook, then i get a validation error, if i look in the gitea webhook add page there is an extra field.

## Screenshots

![Screenshot from 2019-12-26 17-26-33](https://user-images.githubusercontent.com/2660/71482929-80daec80-2805-11ea-9fc1-2b8e424634bc.png)

**Comment by russell** [comment]
after a bit more investigation, it looks like the field is hidden, so you can edit the page with the inspector to work around this for the time being

```
<div class="field" style="display: none;">
			<label>POST Content Type</label>
			<div class="ui selection dropdown" tabindex="0">
				<input type="hidden" id="content_type" name="content_type" value="application/json">
				<div class="default text"></div>
				<i class="dropdown icon" tabindex="0"><div class="menu" tabindex="-1"></div></i>
				<div class="menu" tabindex="-1">
					<div class="item" data-value="1" tabindex="-1">application/json</div>
					<div class="item" data-value="2" tabindex="-1">application/x-www-form-urlencoded</div>
				</div>
			</div>
		</div>
```

**Comment by jvstein** [comment]
Thanks @russell. By setting `value="1"`, taking from a working webhook, I was able to get the form to work.

**Comment by lunny** [comment]
Any one could you send a PR?

**Comment by guillep2k** [comment]
Fixed by #10456
